server/backend/tokens.go

```diff
@@ -12,6 +12,208 @@ import (
+	DEFAULT_OPERATORS                   InterfaceName = "DEFAULT_OPERATORS"
+	GET_APPROVED                        InterfaceName = "GET_APPROVED"
+	GET_CURRENT_FOR                     InterfaceName = "GET_CURRENT_FOR"
+	GRANULARITY                         InterfaceName = "GRANULARITY"
+	HAS_HASH                            InterfaceName = "HAS_HASH"
+	HOLDER_AT                           InterfaceName = "HOLDER_AT"
+	HOLDER_COUNT                        InterfaceName = "HOLDER_COUNT"
+	INCREASE_ALLOWANCE                  InterfaceName = "INCREASE_ALLOWANCE"
+	INCREASE_ALLOWANCE_AND_CALL         InterfaceName = "INCREASE_ALLOWANCE_AND_CALL"
+	INCREASE_SUPPLY                     InterfaceName = "INCREASE_SUPPLY"
+	IS_APPROVED_FOR_ALL                 InterfaceName = "IS_APPROVED_FOR_ALL"
+	IS_HOLDER                           InterfaceName = "IS_HOLDER"
+	IS_OPERATOR_FOR                     InterfaceName = "IS_OPERATOR_FOR"
+	IS_SUPERSEDED                       InterfaceName = "IS_SUPERSEDED"
+	IS_VERIFIED                         InterfaceName = "IS_VERIFIED"
+	MINT                                InterfaceName = "MINT"
+	NAME                                InterfaceName = "NAME"
+	ON_ERC721_RECEIVED                  InterfaceName = "ON_ERC721_RECEIVED"
+	ON_ERC1155_BATCH_RECEIVED           InterfaceName = "ON_ERC1155_BATCH_RECEIVED"
+	ON_ERC1155_RECEIVED                 InterfaceName = "ON_ERC1155_RECEIVED"
+	OPERATOR_BURN                       InterfaceName = "OPERATOR_BURN"
+	OPERATOR_SEND                       InterfaceName = "OPERATOR_SEND"
+	OWNER_OF                            InterfaceName = "OWNER_OF"
+	REMOVE_VERIFIED                     InterfaceName = "REMOVE_VERIFIED"
+	REVOKE_OPERATOR                     InterfaceName = "REVOKE_OPERATOR"
+	SAFE_BATCH_TRANSFER_FROM            InterfaceName = "SAFE_BATCH_TRANSFER_FROM"
+	SAFE_TRANSFER_FROM                  InterfaceName = "SAFE_TRANSFER_FROM"
+	SAFE_TRANSFER_FROM_1                InterfaceName = "SAFE_TRANSFER_FROM_1"
+	SEND                                InterfaceName = "SEND"
+	SET_APPROVAL_FOR_ALL                InterfaceName = "SET_APPROVAL_FOR_ALL"
+	SUPPORTS_INTERFACE                  InterfaceName = "SUPPORTS_INTERFACE"
+	SYMBOL                              InterfaceName = "SYMBOL"
+	TOKENS_RECEIVED                     InterfaceName = "TOKENS_RECEIVED"
+	TOKENS_TO_SEND                      InterfaceName = "TOKENS_TO_SEND"
+	TOKEN_BY_INDEX                      InterfaceName = "TOKEN_BY_INDEX"
+	TOKEN_OF_OWNER_BY_INDEX             InterfaceName = "TOKEN_OF_OWNER_BY_INDEX"
+	TOKEN_URI                           InterfaceName = "TOKEN_URI"
+	TOTAL_SUPPLY                        InterfaceName = "TOTAL_SUPPLY"
+	TRANSFER                            InterfaceName = "TRANSFER"
+	TRANSFER_1                          InterfaceName = "TRANSFER_1"
+	TRANSFER_2                          InterfaceName = "TRANSFER_2"
+	TRANSFER_AND_CALL                   InterfaceName = "TRANSFER_AND_CALL"
```

Comment:
Can we use idiomatic go names for these constants? E.g. `TransferFromAndCall` and `ERC20Burnable`.

Also, are the string values used for anything other than map keys? Iota enums would be more efficient

Reply:
done